carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/R.java

```diff
@@ -130,6 +133,11 @@ public static void reinit() {
         }
     }
 
+    private boolean isParameterBeenSet(Parameter parameter, Properties properties){
```

Comment:
no need this method. you can reuse above:
if (CONFIG.get(key).isEmpty()) {
...
}